src/gui/Src/Utils/StringUtil.cpp

```diff
@@ -224,4 +226,23 @@ QString ToDateString(const QDate & date)
         "Dec"
     };
     return QString().sprintf("%s %d %d", months[date.month() - 1], date.day(), date.year());
+    */
+    return QLocale(QString(currentLocale)).toString(date);

```

Comment:
In fact it is also used in the thread creation time, that's why I want locale-dependent date formatting.